stdlib/seq.mc

```diff
@@ -1,9 +1,15 @@
 include "option.mc"
 
-let head = lam s. nth s 0
-let tail = lam s. slice s 1 (length s)
 let null = lam seq. eqi 0 (length seq)
 
+let slice = lam seq. lam n. lam m.
```

Comment:
Maybe I am being too pendantic, but could it be an idea to rename the `n` and `m` arguments to `off` (or `offset`) and `cnt` (or `count`) respectively? It would aid readability since `n` and `m` does not tell me very much about what those arguments represent.

Reply:
Sure!